src/components/tree/treeButtons.tsx

```diff
@@ -35,12 +37,16 @@ export function TreeButtons(props: Props): ReactElement {
         <FaSearchMinus/>
       </button>
 
-      <button style={activeZoomButton ? selectedButtonStyles : unselectedButtonStyles} onClick={zoomToSelected}>
-        {props.t("Zoom to Selected")}
-      </button>
+      {showZoomButton && (
```

Comment:
> [@victorlin] It might make more sense to use this feature to replace the existing "Zoom to selected" button, rather than adding it to the "Focus on selected" toggle.

I think it's a toggle state not a button action, as I like the dynamic behaviour as you change the date slider / filters. But you're right that they are similar, which is why I got rid of the "zoom to selected" button when in focus mode!

Reply:
Ah I see, thanks for the examples in the review app. I was mixing up your idea of "dynamic zoom" with the existing "zoom to selected" button.